plugins/tours/tests/phpunit/unit/_renderCustomColumnContent.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for _render_custom_column_content().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Mockery as m;
+use Brain\Monkey;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\_render_custom_column_content;
+
+/**
+ * Class Tests_RenderCustomColumnContent
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Unit
+ * @group   tours
+ * @group   admin
+ */
+class Tests_RenderCustomColumnContent extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var Mockery
+	 */
+	protected $post;
+
+	/**
+	 * The post ID
+	 *
+	 * @var integer
+	 */
+	protected $ID;
+
+	/**
+	 * The post type
+	 *
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/admin/wp-list-table.php';
+
+		$this->post = m::mock( 'post' );
+		$this->post->ID = (int) 99;
+		$this->post->post_type = 'tours';
+	}
+
+	/**
+	 *  Test _render_custom_column_content() should echo 'tour_id' when evaluating $column_name.
+	 */
+	public function test_should_echo_tour_id_when_evaluating_column_name() {
+		Monkey\Functions\expect( '_render_custom_column_content' )
+			->once()
+			->with( 'column_name', 'tour_id' )
+			->andReturn( 99 );
+		$expected = $this->post->ID;
+
+		$this->assertSame( $expected, _render_custom_column_content( $column_name, $this->post->ID ) );
+	}
```

Comment:
This test is validating the tour ID is echoed, i.e. rendered. That means it should send it out to the browser. How can you test this?

```php
public function testShouldEchoTourIdWhenColumnNameIsTourId() {
	$tour_id = 99;

	Monkey\Functions\expect( 'get_post_meta' )->with( $tour_id, 'tour_year', true )->never();
	Monkey\Functions\expect( 'get_post_field' )->with( 'menu_order', $tour_id )->never();

	$this->expectOutputString( $tour_id );
	_render_custom_column_content( 'tour_id', $tour_id );
}
```

Here are the [docs on `expectOutputString`](expectOutputString).

Reply:
@hellofromtonya Could you please post that doc link again to `expectOutputString`? It didn't get passed in the first time.

Reply:
Doh. Here it is https://phpunit.readthedocs.io/en/9.0/writing-tests-for-phpunit.html#testing-output.

Reply:
@hellofromtonya Updated per above.